rotors_gazebo_plugins/src/gazebo_wind_plugin.cpp

```diff
@@ -208,6 +331,109 @@ void GazeboWindPlugin::CreatePubsAndSubs() {
                                            true);
 }
 
+void GazeboWindPlugin::ReadCustomWindField(std::string& custom_wind_field_path) {
+  std::ifstream fin;
+  fin.open(custom_wind_field_path);
+  if (fin.is_open()) {
+    std::string data_name;
+    float data;
+    // Read the line with the variable name.
+    while (fin >> data_name) {
+      // Save data on following line into the correct variable.
```

Comment:
The two alternatives I can think of are either using an `enum`, or writing a function that parses each string to an integer using also `if` and `else if` statements. Which one do you prefer? Did you have any better idea in mind?